source/CellMeasurer/CellMeasurer.js

```diff
@@ -1,35 +1,42 @@
 /** @flow */
-import { PureComponent } from 'react'
+import React from 'react'
 import { findDOMNode } from 'react-dom'
+import CellMeasurerCache from './CellMeasurerCache.js';
+
+type Children = (params: { measure: () => void }) => React.Children
```

Comment:
Yeah, Flow doesn't support pulling in types this way yet (although I think they're planning to add this soon!)

In this case, I think `React.Element<any>` yeah